test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -2284,6 +2284,24 @@ public void ValidateReferencesAllowsPartialFileNames()
             Assert.True(true);
         }
 
+        [Fact]
+        public void ValidateReferencesAllowsNullFramework()
+        {
+            // Arrange
+            var files = new[] {
+                        new PhysicalPackageFile { TargetPath = @"lib" + Path.DirectorySeparatorChar + "net40" + Path.DirectorySeparatorChar + "foo.dll" },
+                        new PhysicalPackageFile { TargetPath = @"lib" + Path.DirectorySeparatorChar + "net40" + Path.DirectorySeparatorChar + "bar.dll" },
+                        new PhysicalPackageFile { TargetPath = @"lib" + Path.DirectorySeparatorChar + "net40" + Path.DirectorySeparatorChar + "baz.exe" },
+                    };
+            var packageAssemblyReferences = new PackageReferenceSet((NuGetFramework)null, new string[] { "foo.dll", "bar", "baz" });
+
+            // Act and Assert
+            PackageBuilder.ValidateReferenceAssemblies(files, new[] { packageAssemblyReferences });
```

Comment:
What's the nuspec equivalent here?

Reply:
The nuspec path does hit it, I just don't know what scenario validate reference assembly is in this case.

Reply:
https://github.com/cryptlex/lexactivator-dotnet/blob/master/src/Cryptlex.LexActivator/Cryptlex.LexActivator.nuspec#L23 this is the nuspec that triggered the bug report